third_party/move/move-compiler-v2/src/bytecode_generator.rs

```diff
@@ -1795,10 +1799,14 @@ impl<'env> Generator<'env> {
         }
         match pat {
             Pattern::Wildcard(id) => {
-                let pat_ty = self.env().get_node_type(*id);
                 if !match_mode.is_probing() {
-                    let temp = self.new_temp(pat_ty);
-                    self.emit_assign_with_convert(*id, temp, values[0], AssignKind::Inferred)
+                    // Maybe do a conversion, but otherwise drop the value.
```

Comment:
Could not immediately figure out if there was already a test exercising this. Can you point it to me, please? If not, please add one.